ZippyIdSDK/ZippyIdSDK/Network/ApiClient.swift

```diff
@@ -61,24 +62,23 @@ class ApiClient {
             .request(request: request)
             .transformed(with: { (data) -> String in
                 let json: [String: String] = try JSONSerialization.jsonObject(with: data, options: []) as! [String: String]
-                return json["request_id"]!
+                return json["verification_id"]!
             })
     }
     
     func getJobStatus(customerId: String) -> Future<ZippyResult> {
         let url: URL = URL(string: baseUrl)!
             .appendingPathComponent("v1")
-            .appendingPathComponent("result")
+            .appendingPathComponent("verification")
         let params: [String: String] = [
-            "customer_uid": customerId,
-            "secret_key": secret,
-            "api_key": key,
+            "customer_uid": customerId
         ]
         
         var urlComponents = URLComponents(url: url, resolvingAgainstBaseURL: false)!
         urlComponents.queryItems = params.map { URLQueryItem(name: $0.key, value: $0.value) }
         var request: URLRequest = URLRequest(url: urlComponents.url!)
         request.httpMethod = "GET"
+        request.addValue("Token token=" + apiKey, forHTTPHeaderField: "Authorization")
```

Comment:
Nevajag kkādu getURLRequest, kur var iebarot paramus un šis būtu uzstādīts pēc default? Jo jau 2 reizi redzu token uzsetošanu